webapps/camunda-commons-ui/lib/widgets/selection-type/cam-widget-selection-type.template.html

```diff
@@ -0,0 +1,50 @@
+<div class="form-inline">
+  <div>
+    <label for="instance"></label>
+    <input type="radio"
+           id="instance"
+           name="selection_type"
+           value="INSTANCE"
+           ng-model="selectionType"
+           ng-change="updateSelectionType(selectionType)" />
+    {{ 'INSTANCE' | translate }}
+    <span class="glyphicon glyphicon-question-sign"
+          uib-tooltip="{{ 'CAM_WIDGET_SELECTION_TYPE_INSTANCE_TOOLTIP' | translate }}"
+          tooltip-placement="right"></span>
+  </div>
+  <div>
+    <label for="query"></label>
+    <input type="radio"
+           id="query"
+           name="selection_type"
+           value="QUERY"
+           ng-model="selectionType"
+           ng-change="updateSelectionType(selectionType)" />
+    {{ 'QUERY' | translate }}
+    <span class="glyphicon glyphicon-question-sign"
+          uib-tooltip="{{ 'CAM_WIDGET_SELECTION_TYPE_QUERY_TOOLTIP' | translate }}"
+          tooltip-placement="right"></span>
+  </div>   
+</div>
+
+<div class="instance"
+     ng-if="selectionType === 'INSTANCE' && toggleState === 'on' && totalInstancesCount > 12">
```

Comment:
Can we make the check `totalInstancesCount > 12` dependent on the actual page size? If we ever change the page size or make it configurable, this logic breaks silently.